### ISSUE TITLE
Settings dialog: pages are not sorted according to their sortorder index

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Risehat/freemedforms#190]
```
To code
```

Original issue reported on code.google.com by `eric.mae...@gmail.com` on 14 Oct 2012 at 9:02

**Comment by GoogleCodeExporter** [comment]
```
Please elaborate: where is the exact problem? What did you find ;-)
When I change the sortOrder() return value, the order changes in the right 
direction too...
Can you give an example?
```

Original comment by `christian.a.reiter@gmail.com` on 14 Oct 2012 at 6:05

**Comment by GoogleCodeExporter** [comment]
```
Well it looks like the order of the settings pages are wrong. Expected:
- General
- Patients
- Forms
- PMHx
- Alerts
- Drugs
- Printing
- Templates
- Accountancy
```

Original comment by `eric.mae...@gmail.com` on 14 Oct 2012 at 9:02

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision c0a94eab4ea1.
```

Original comment by `eric.mae...@gmail.com` on 23 Dec 2012 at 1:01
- Changed state: **Fixed**
